src/ClientData/include/ClientData/TrackData.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_TRACK_DATA_H_
+#define CLIENT_DATA_TRACK_DATA_H_
+
+#include <absl/synchronization/mutex.h>
+
+#include "ClientData/TimerChain.h"
+
+namespace orbit_client_data {
+
+class TrackData {
```

Comment:
I understand that this is the current state of the code, but in the future, Track only will have a bunch of Panes which therefore will have the Events (Timers/Samples), do we agree on that?

Reply:
This structure is not going to be used by CaptureDataProvider, this is just a container for Legacy TimerInfo stuff.